libraries/Swagger/ConnectorAPI.json

```diff
@@ -1761,10 +1761,6 @@
                     "description": "Longitude of the location [WGS 84](https://en.wikipedia.org/wiki/World_Geodetic_System)",
                     "type": "number"
                 },
-                "type": {
```

Comment:
>type [](start = 17, length = 4)

No, this file shouldn't be hand edited, it is sourced from intercom project

Reply:
Will revert this changes.